MaterialLib/MPL/PropertyType.h

```diff
@@ -287,7 +287,7 @@ static const std::array<std::string, PropertyType::number_of_properties>
                              "specific_heat_capacity",
                              "storage",
                              "thermal_conductivity",
-                             "thermal_expansivity",
+                             "thermal_expansion",
```

Comment:
@wenqing @scinopode please discuss whether all of the items should be called one or the other way.
In my opinion expansivity is better because it describes how something would change, whereas expansion describes the actual change.

Reply:
@endJunction Forgot to add `_coefficent`. Agree with you.  Will drop this change.